plugins/modules/elb_classic_lb_info.py

```diff
@@ -136,13 +348,20 @@
 from ansible_collections.amazon.aws.plugins.module_utils.retries import AWSRetry
 from ansible_collections.amazon.aws.plugins.module_utils.tagging import boto3_tag_list_to_ansible_dict
 
-from ansible_collections.community.aws.plugins.module_utils.modules import AnsibleCommunityAWSModule as AnsibleAWSModule
+from ansible_collections.amazon.aws.plugins.module_utils.modules import AnsibleAWSModule
+
 
-MAX_AWS_RETRIES = 5
-MAX_AWS_DELAY = 5
+def list_elbs(connection: Any, load_balancer_names: List[str]) -> List[Dict]:
```

Comment:
Have you tried using  botocore.client.BaseClient for connection type hinting?
```suggestion
def list_elbs(connection: botocore.client.BaseClient, load_balancer_names: List[str]) -> List[Dict]:
```

Reply:
There's a retry decorator applied to connection, it's not a BaseClient.  Technically it's a RetryingBotoClientWrapper.

We should probably discuss how to handle type hints for botocore clients/resources in a broader forum.

Reply:
keeping it `Any` for now.